zbus_macros/tests/tests.rs

```diff
@@ -312,6 +312,68 @@ fn test_interface() {
     }
 }
 
+#[test]
+fn test_multi_interface_sync() {
```

Comment:
Kudos for not only adding a unit test but also a more e2e test :+1: but let's instead edit [our zbus e2e test](https://github.com/dbus2/zbus/blob/main/zbus/tests/e2e.rs) to add a write-only property to the interface there and the proxy in there to test it. Reminder to self: that test file really needs to be split.